encrypted-media-respec.html

```diff
@@ -2301,9 +2303,92 @@
             <li><p>Resolve <var>promise</var>.</p></li>
           </ol>
         </section>
-      </section>
 
-      <section>
+        <section id="monitor-cdm">
+          <h4>Monitor for CDM Changes</h4>
+          <p>
+            The Monitor for CDM Changes algorithm executes steps required when various aspects of CDM state change. It is run for each
+            <a>MediaKeySession</a> object in parallel to the main event loop but not in parallel to other procedures defined in this
+            specification that are run in parallel to the main event loop.
+          </p>
```

Comment:
Should we note that this algorithm only applies to changes that are not covered by other algorithms? IOW, it does not apply to changes that are directly related to an application action. For example, `update()` may result in messages, key status change, and expiration change, but those are all handled within that algorithm.